fastlane/lib/fastlane/actions/spm.rb

```diff
@@ -50,6 +51,11 @@ def self.available_options
                                        description: "Enables code coverage for the generated Xcode project when using the 'generate-xcodeproj' and the 'test' command",
                                        type: Boolean,
                                        optional: true),
+          FastlaneCore::ConfigItem.new(key: :parallel,
+                                       env_name: "FL_SPM_PARALLEL",
+                                       description: "Enables running tests in parallel when using the 'test' command",
+                                       type: Boolean,
+                                       optional: true),
```

Comment:
Since this is a boolean, I'd suggest making it non-optional and defaulting it to false (to keep the previous behavior; in favor of backwards compatibility):

```suggestion
                                       type: Boolean,
                                       default_value: false),
```